config/config.yaml

```diff
@@ -74,6 +76,8 @@ operations:
       limit: 1000
 
 services:
+  schedulerManager:
+    defaultPdbMaxUnavailable: "5%"
```

Comment:
what is this for? from the doc, it says default will be 10%

> *pdbMaxUnavailable**: Defines the disruption budget for game rooms. Optional and defaults to 10%.

also, whats the difference between this and 

```
scheduler:
        maxUnavailable: 10%
```

Reply:
scheduler.maxUnavailable is garbage that I forgot to remove. I'll update the docs with the 5% default